File: java to c++/DirlistRequest.cpp
#include "DirlistRequest.h"

namespace bgu
{
	namespace spl171
	{
		namespace net
		{
			namespace impl
			{
				namespace TFTP
				{
					namespace Commands
					{
						using Response = bgu::spl171::net::impl::TFTP::Commands::Responses::Response;

						short DirlistRequest::getOpCode()
						{
							return OpCode::Dirlist::getValue();
						}
					}
				}
			}
		}
	}
}

File: spl-net/src/main/java/bgu/spl171/net/impl/TFTP/Commands/DirlistRequest.java
package bgu.spl171.net.impl.TFTP.Commands;

import bgu.spl171.net.impl.TFTP.Commands.Responses.Response;

public class DirlistRequest implements Command {
    @Override
    public short getOpCode() {
        return OpCode.Dirlist.getValue();
    }

    @Override
    public Response execute() {
        return null;
    }
}
